impeller/renderer/renderer_dart_unittests.cc

```diff
@@ -85,3 +93,10 @@ TEST_P(RendererDartTest, CanRunDartInPlaygroundFrame) {
 
 }  // namespace testing
 }  // namespace impeller
+
+int main(int argc, char** argv) {
```

Comment:
Instead of installing your own `main`, use `run_all_unittests.cc` from `//flutter/testing:testing`. It does all the usual setup you want :)

Reply:
Oh whoops, looks like I pulled in `testing_lib` instead of `testing` for some reason. Done.